src/udev/udev-builtin-blkid.c

```diff
@@ -293,6 +298,9 @@ static int builtin_blkid(sd_device *dev, sd_netlink **rtnl, int argc, char *argv
         blkid_probe_set_superblocks_flags(pr,
                 BLKID_SUBLKS_LABEL | BLKID_SUBLKS_UUID |
                 BLKID_SUBLKS_TYPE | BLKID_SUBLKS_SECTYPE |
+#if defined(BLKID_SUBLKS_FSINFO)
```

Comment:
we more often spell this as `#ifdef …` instead of `#if defined(…)`, but I guess this works too

Reply:
No problem, will change to `#ifdef` to stay consistent

Reply:
> we more often spell this as `#ifdef …` instead of `#if defined(…)`, but I guess this works too

@poettering I tried to count `#ifdef` and `#if defined`, but our code seem to have much `#if defined` than `#ifdef`. What's the difference and why `#ifdef` is a preferable way? Though, I do not have strong opinion about that. That's just a pure question.

Reply:
We usually use `#if defined(…)` if we have more complex expressions, no? (because you have to there...) otherwise it's more `#ifdef`...

Reply:
but i don't think this really matters, at all